pkg/cmd/ssh/ssh_test.go

```diff
@@ -635,19 +661,28 @@ var _ = Describe("SSH Command", func() {
 
 	Describe("ValidArgsFunction", func() {
 		BeforeEach(func() {
+			testMachine2 := &machinev1alpha1.Machine{
```

Comment:
things to improve:
- rename testMachine to monitoringMachine to be consistent with the other naming schema like (monitoringNode and workerNode)
- there is a workerNode but it's name is monitoring2-node. This is inconsistent
- there is also a case where the user does not have access to the seed, this case also has to be tested
- Also make sure you have a node in your test that has a different name (like workerNode), currently you only have node names starting with monitoring

Reply:
> here is also a case where the user does not have access to the seed, this case also has to be tested

I tried to do that before, but I don't know how to let `err` return is forbidden. 

Now I am using the exits object `testMachine, testNode, monitoringMachine` to test the case, instead of using a new `workNode`. to make more clearly [5f5828d](https://github.com/gardener/gardenctl-v2/pull/347/commits/5f5828d857b668d3e0e8f7b463efb31c04787ba2)

Reply:
switch PR to draft status, I might find the solution.

Reply:
Now, done ~~  e8436ee91a6af7aeaf9820c20532614cea6b86d2